Doc/library/stdtypes.rst

```diff
@@ -1796,7 +1796,15 @@ expression support in the :mod:`re` module).
 
    Return ``True`` if the string is a titlecased string and there is at least one
    character, for example uppercase characters may only follow uncased characters
-   and lowercase characters only cased ones.  Return ``False`` otherwise.
+   (i.e. all that are not cased) and lowercase characters only follow cased [4]_ ones.
+   Return ``False`` otherwise.
+
+   Also see :meth:`title`.
+
+   .. note::
+
+      This method is different from the standard publishing definition
+      of a title cased text fragment in which minor words would not be capitalized [6]_.
```

Comment:
nit: currently 'titlecased' is used instead of 'title cased' (or 'title-cased' for that matter), so I think picking either one and making it consistent throughout would be good.